app/javascript/components/widgets/twitter/panel.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import gql from 'graphql-tag';
+import { Query } from 'react-apollo';
+import { LoadingMessage, DisconnectedMessage } from '@messages/message';
+import styled from 'styled-components';
+import { colors, fontSizes } from '@lib/theme';
+import { parseMonthDate } from '../../../lib/datetime';
+import TwitterProfile from './tweet_profile';
+import TweetStats from './tweet_stats';
+import TweetBody from './tweet_body';
```

Comment:
This should probably use absolute imports
`import TweetBody from '@twitter/tweet_body';`